src/main/java/com/online/store/controllers/shoppingcart/controller/DeleteProductFromShoppingCartController.java

```diff
@@ -21,7 +21,9 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp)
             throws IOException {
         Long productId = Long.valueOf(req.getParameter("id"));
         shoppingCartService.deleteProduct(
-                shoppingCartService.getByUserId(USER_ID), productService.getById(productId));
+                shoppingCartService.getByUserId(
+                        (Long) req.getSession().getAttribute(USER_ID)),
+                productService.getById(productId));
```

Comment:
local variables would make the code more readable here

Reply:
done)

Reply:
done